app/views/admin/PageClanEdit.vue

```diff
@@ -0,0 +1,95 @@
+<script>
+import { getClan } from 'core/api/clans'
+import CocoCollection from 'app/collections/CocoCollection'
+import Clan from 'app/models/Clan'
+
+const api = require('core/api')
+require('core/services/filepicker')()
```

Comment:
Is this needed here, or is it taken care of [when an ImageFileTreema is created](https://github.com/codecombat/ozaria/blob/4478a19c21224f0053f6bca4458635bbf2549648/app/core/treema-ext.coffee#L238) anyway?

Reply:
Excellent catch. Will remove and double check functionality.

I can think of some other places then we are double initializing in internal tools.
Thanks!

Reply:
Tested and can be removed